Package.swift

```diff
@@ -302,6 +303,20 @@ var targets: [Target] = [
             .define("MOCKING", .when(configuration: .debug)),
         ]
     ),
+    .target(
+        name: "TuistApp",
```

Comment:
Even though we named the internal repository `app`, I think `server` is more descriptive here from the CLI standpoint:

```suggestion
        name: "TuistServer",
```